CHANGES.md

```diff
@@ -1,50 +1,58 @@
 # Changes
 
+## Version 0.7.9
+
+- wally_is_elements_build now takes a size_t output instead of uin64_t.
+
+- elements_pegout_script_from_bytes, asset_pak_whitelistproof and
+  psbt_to_bytes now follow the library convention for too-short buffers
+  instead of returning WALLY_EINVAL. See the generated API documentation
+  section "Variable Length Output Buffers" for details.
+
+- psbt_combine has been changed to only combine one PSBT into another.
+
+- psbt_to_bytes, psbt_get_length and psbt_to_base64 now take a flags argument.
+
+- FINGERPRINT_LEN was renamed to BIP32_KEY_FINGERPRINT_LEN for
+  consistency - You should change any references in your source when upgrading.
+
+- The following PSBT functions have been renamed for consistency:
+  - wally_add_new_keypath -> wally_keypath_map_add
+  - wally_add_new_partial_sig -> wally_partial_sigs_map_add
+  - wally_add_new_unknown -> wally_unknowns_map_add
+  - wally_extract_psbt -> wally_psbt_extract
+  - wally_finalize_psbt -> wally_psbt_finalize
+  - wally_sign_psbt -> wally_psbt_sign
+  - wally_combine_psbts -> wally_psbt_combine
+
 ## Version 0.7.8
 
-- Deprecate Python 2 wheels
-- Honor CFLAGS, default to O2
-- Bug fixing
+- Python 2 wheels are now deprecated. Users should move to Python 3 as soon as possible.
 
 ## Version 0.7.7
 
-- Expose wally_psbt_get_length
```

Comment:
Why CHANGES.md should not include API changes that does not break the API?
I would expect to find the addition of new exposed functions listed here.
Do we want to do this to keep such file shorter?

Reply:
@LeoComandini  yes, the idea here is to only list the changes that users might need to account for when upgrading, and we don't want these to get lost in the noise when we add new calls. If there are no API changes in CHANGES.md when upgrading, we want users to be able to tell that at a glance.

Users can get general info about a release from the release page, git history, generated docs, and the release announcement, all of which are better placed to detail new features etc.